### PR TITLE
Added support for suggest an edit and page history

### PR DESCRIPTION
**Pull request by vikram-redhat**
When merged, this will remain in the master branch only. This doesn't need to get CPed anywhere.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@openshift/team-documentation - PTAL. 

* The template now suggests that users only submit docs related issues and if they want to contribute they can do so via a PR. This is left as a comment that doesn't appear in the final submitted issue.

**Comment by bfallonf** [comment]
@vikram-redhat LGTM :ship:

**Comment by gaurav-nelson** [comment]
Thanks @vikram-redhat LGTM 🏭

**Comment by kalexand-rh** [comment]
I'd switch "only" and "submit."
Can we encode links in the comment? If so, I'd add a link to the the doc contribution guidelines and a link to the process for submitting a dev bug, if there's one that we can share.

**Comment by vikram-redhat** [comment]
@kalexand-rh - hey. The text that is displayed is in Markdown format. Although I can add a link, it will just come up as text and not a clickable link unless the user looks at the preview.

**Comment by kalexand-rh** [comment]
@vikram-redhat, thanks! It's your call - I'd probably add the unformatted links. (I worry about people finding information on their own.)

File: issue_template.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<!--
+Please only submit documentation-related issues with this form, or follow the contributing to OpenShift guidelines to submit a PR.
```
Comment (ahardin-rh) [inline issue_template.md]: Contribute to OpenShift Documentation guidelines

**Comment by ahardin-rh** [comment]
Just one comment from me. Otherwise, looks good! :star2:

File: issue_template.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+-->
```
Comment (mburke5678) [inline issue_template.md]: s/Please only submit/Please submit only

Comment (mburke5678) [inline issue_template.md]: Do we want a link with _contributing to OpenShift guidelines_ 
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc

**Comment by mburke5678** [comment]
@vikram-redhat A couple of nits

**Comment by tmorriso-rh** [comment]
The PR title field reads "[online] Edit Suggested..." 

s/Edit Suggested/Edit suggested

File: issue_template.md (hunk 2 above)
Comment (tmorriso-rh) [reply issue_template.md]: +1

**Comment by vikram-redhat** [comment]
Ok all issues addressed. Will give it one more day for any more feedback and then merge.

**Comment by vikram-redhat** [comment]
xref: removing revision histories from the 3.9 branch: https://github.com/openshift/openshift-docs/pull/8389

**Comment by bmcelvee** [comment]
@vikram-redhat LGTM :four_leaf_clover:

**Pull request merged**